www/js/pages/account-categorisation/account-categorisation.controller.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * @name account categorisation controller
+ * @author Nazanin Reihani Haghighi
+ * @contributors []
+ * @since 06/17/2018
+ * @copyright Binary Ltd
+ */
+
+(function() {
+    angular
+        .module("binary.pages.account-categorisation.controllers")
+        .controller("AccountCategorisationController", AccountCategorisation);
+
+    AccountCategorisation.$inject = ["$scope", "$state", "$timeout", "$translate", "websocketService", "accountService", "clientService", "alertService"];
+
+    function AccountCategorisation($scope, $state, $timeout, $translate, websocketService, accountService,
+        clientService, alertService) {
+        const vm = this;
+        let loginid = '';
+        const params = {
+            request_professional_status: 0,
+            tax_identification_number  : '',
+            tax_residence              : ''
+        };
+
+        const hasValueIn = (arr, val) => _.indexOf(arr, val) > -1;
+        const isLandingCompanyOf = (targetLandingCompany, accountLandingCompany) =>
+            clientService.isLandingCompanyOf(targetLandingCompany, accountLandingCompany);
+
+        $scope.$on('set-settings:error', (e, error) => {
+            if (error.code === 'InputValidationFailed') {
+                $state.go('profile');
+            } else {
+                alertService.displayError(error.message);
+            }
+        });
+
+        $scope.$on('set-settings', (e, error) => {
```

Comment:
@nazaninreihani 
it's better to name the second param to something like `response` or `message` instead of `error`.